src/config.c

```diff
@@ -214,5 +368,10 @@ void free_config(void)
 
 	free(ini_config->smtp_config);
 	free(ini_config->postgres_config);
+	free(ini_config->general_config);
 	free(ini_config);
+	free(log_filepath);
+	if (log_file != NULL) {
+		fclose(log_file);
+	}
```

Comment:
Is this going to become more and more frequent you believe? Should we just make it a function right now to avoid patching them later on?

Reply:
It depends. If every database system opens and closes the file then yes this is going to be in every database we support. If we aim to keep the file open and have all the database systems use a global variable to `FILE *` then having this check once, should be ok.